src/apps/common/submodules/dragableUploads/dragableUploads.scss

```diff
@@ -0,0 +1,160 @@
+@import '../../../../css/partials/base';
+
+.dragable-wrapper {
```

Comment:
Please use a more specific selector here, usually it should contain the name of the common control.